test/loam.spec.js

```diff
@@ -395,11 +441,14 @@ describe('Given that loam exists', () => {
                 .then((ds) => ds.bytes()) // Call an accessor to trigger operation execution
                 .then(
                     (result) => {
-                        throw new Error('render() promise should have been rejected but got ' +
-                            result + ' instead.'
+                        throw new Error(
+                            'render() promise should have been rejected but got ' +
```

Comment:
I added format-on-save to my editor since the last time touching this file and there was some minor fallout.